### ISSUE TITLE
jQuery.fn.v_toggleVolume() should be jQuery.fn.v_toggle_volume()

### ISSUE THREAD

**Issue by alessandro-newzoo** [pupunzi/jquery.mb.vimeo_player#26]
Hi,

Based on the docs, the method **jQuery.fn.v_toggleVolume()** should toggle the mute status, but if I use that the console will say that **.v_toggleVolume is not a function**

So I took a look at the source files and turns out we should use **jQuery.fn.v_toggle_volume** instead, which in fact works fine.

Not working example (based on the Docs):

```
	$('.icon-unmute').click(function() {
		$('#P1').v_toggleVolume();
	});
```

Working example (not in the Docs, based on source files inspection):

```
	$('.icon-unmute').click(function() {
		$('#P1').v_toggle_volume();
	});
```

So long story short, please fix the docs :)

Thanks a lot!
